### PR TITLE
Added NNCF actions for MobileNet model

### PR DESCRIPTION
**Pull request by pfinashx**
Added loading AUX weights for NNCF and MobileNet model

### CODE REVIEW

**Comment by nervana-ff** [comment]
Can one of the admins verify this patch?

File: tests/test_ote_training.py [hunk 1]
```diff
@@ -271,6 +276,57 @@ def _run_ote_training(self, data_collector):
...
+
+        logger.info("Creating NNCF task and structures")
+        self.nncf_model = ModelEntity(
+            dataset,
+            self.environment_for_nncf.get_model_configuration(),
+        )
+        self.nncf_model.set_data("weights.pth", trained_model.get_data("weights.pth"))
+        
+        if 'aux_model_1.pth' in trained_model.model_adapters:
+            self.nncf_model.set_data("aux_model_1.pth", trained_model.get_data("aux_model_1.pth"))
```
Comment (LeonidBeynenson) [inline tests/test_ote_training.py:305]: @pfinashx , are you sure that only this name is always used?

**Review by LeonidBeynenson — COMMENTED** [review]
Almost LGTM

File: tests/test_ote_training.py (hunk 1 above)
Comment (pfinashx) [reply tests/test_ote_training.py:305]: Yes, at least for Mobilenet models.

Comment (pfinashx) [reply tests/test_ote_training.py:305]: Yep, at least for Mobilenet model

**Review by LeonidBeynenson — APPROVED** [review]

**Comment by LeonidBeynenson** [comment]
@sovrasov , please review it. It seems to me that it is OK.

**Review by sovrasov — APPROVED** [review]

**Pull request merged**